app.js

```diff
@@ -16,3 +16,393 @@ const navSlide = () => {
+  },
+  {
+    name: 'Website Portfolio',
+    description:
+      "A daily selection of privately personalized reads; no accounts or sign-ups required. Has been the industry's standard.",
+    workStack: ['HTML', 'Bootstrap', 'Ruby'],
+  },
+  {
+    name: 'Profesional Art Printing Data More',
+    description:
+      "A daily selection of privately personalized reads; no accounts or sign-ups required. Has been the industry's standard.",
+    workStack: ['HTML', 'Bootstrap', 'Ruby'],
+  },
+  {
+    name: 'Data Dashboard Healthcare',
+    description:
+      "A daily selection of privately personalized reads; no accounts or sign-ups required. Has been the industry's standard.",
+    workStack: ['HTML', 'Bootstrap', 'Ruby'],
+  },
+  {
+    name: 'Website Portfolio',
+    description:
+      "A daily selection of privately personalized reads; no accounts or sign-ups required. Has been the industry's standard.",
+    workStack: ['HTML', 'Bootstrap', 'Rub000y'],
+  },
+];
+
+if (portfolio) {
+  const div = document.createElement('div');
+  const img = document.createElement('img');
+  div.classList.add();
+  projectsWrapperDiv.classList.add('recent-project-wraper', 'project-block');
+  mostRecentProjectWrapper.classList.add('most-recent-project-wraper');
+  descktopProjectsWrapperDiv.classList.add(
+    'desktop-recent-project-wraper',
+    'project-block',
+  );
+
+  img.setAttribute('src', `${recentProjects[0].image}`);
+  img.setAttribute('alt', 'recent work');
+  img.classList.add('recent-image');
+  div.classList.add('most-recent-work-project-details-waraper');
+  div.innerHTML = `<h3 class="recent">${recentProjects[0].name}</h3>
+            <p class="recent-project-details">
+              ${recentProjects[0].description}
+            </p>
+            <ul class="recent-buttons-container">
+              <li>
+                <a
+                  href="https://www.w3.org/Style/CSS/"
+                  class="recent-button active-btn-effect hover-btn-effect"
+                  id="recent-button-1"
+                  >CSS</a
+                >
+              </li>
+              <li>
+                <a
+                  href="https://html.com/"
+                  class="recent-button active-btn-effect hover-btn-effect"
+                  id="recent-button-2"
+                  >HTML</a
+                >
+              </li>
+              <li>
+                <a
+                  href="
+ https://getbootstrap.com/"
+                  class="recent-button active-btn-effect hover-btn-effect"
+                  id="recent-button-3"
+                  >Bootstrap</a
+                >
+              </li>
+              <li>
+                <a
+                  href="https://www.ruby-lang.org/en/"
+                  class="recent-button active-btn-effect hover-btn-effect"
+                  id="recent-button-4"
+                  >Ruby</a
+                >
+              </li>
+            </ul>
```

Comment:
- [ ] 👏 Great job on adding the project technology list in the project card! It's a valuable addition to showcase the technologies used in each project. However,  I noticed that you are currently limiting the display to only three static technologies. In a real-life scenario, projects can have varying numbers of technologies, and it's essential to display them all. To improve this, I recommend making the technology list in the popup modal dynamic by using a loop. This way, you can display all the technologies for each project, regardless of their number. It will provide a more accurate representation of the technologies used in each project and enhance the user experience.

Keep up the good work and consider implementing this dynamic approach to showcase the technologies in the popup modal! 🚀💻🌟